src/boot/sign.c

```diff
@@ -0,0 +1,416 @@
+/* SPDX-License-Identifier: LGPL-2.1-or-later */
+
+#include <getopt.h>
```

Comment:
hmm, i think we should try harder to name the entrypoint .c file like the binary they result in. i.e. rename sign.c to sbsign.c please